kong/plugins/datadog/handler.lua

```diff
@@ -120,6 +122,13 @@ function DatadogHandler:log(conf)
 
   local queue_id = get_queue_id(conf)
   local q = queues[queue_id]
+  if q and not tx_deepcompare(conf, q.conf) then
```

Comment:
have we consider using a `crud` event handler to flush queue
upon plugin config change instead of doing deepcompare on
every request?

Reply:
It also seems to be imprudent to copy and paste this logic into every plugin that uses queues.